gengokit/gengokit_test.go

```diff
@@ -172,7 +168,7 @@ func TestApplyTemplateFromPath(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	te, err := newTemplateExecutor(dt, goImportPath)
+	te, err := newTemplateExecutor(dt, goImportPath, goImportPath)
```

Comment:
@hasIan I wanna say that I'm torn, but I can't actually seem to form a concrete idea about why. I think its probably fine to use the zero value to default the path. So long as the function has a comment stating the behavior (for godoc)